nano/lib/logger_mt.hpp

```diff
@@ -134,6 +134,6 @@ class logger_mt
 private:
 	nano::mutex last_log_time_mutex;
 	std::chrono::steady_clock::time_point last_log_time;
-	boost::log::sources::severity_logger_mt<severity_level> boost_logger_mt;
+	boost::log::sources::severity_logger<severity_level> boost_logger_mt;
```

Comment:
This version of logger (without _mt) is not thread safe and we're using it from multiple threads.